query.go

```diff
@@ -37,6 +37,7 @@ func (fb *Firebase) EndAt(value string) *Firebase {
 func (fb *Firebase) OrderBy(value string) *Firebase {
 	c := fb.copy()
 	if value != "" {
+		value = "\"" + value + "\""
```

Comment:
> stylistic nit: ...

I would lean towards using backticks instead of using the `fmt` package since we don't need very specific formatting